src/block/block_open.c

```diff
@@ -82,33 +82,31 @@ __wt_block_manager_create(WT_SESSION_IMPL *session, const char *filename, uint32
 }
 
 /*
- * __block_destroy --
- *     Destroy a block handle.
+ * __wt_block_close --
+ *     Close a block handle.
  */
-static int
-__block_destroy(WT_SESSION_IMPL *session, WT_BLOCK *block)
+int
+__wt_block_close(WT_SESSION_IMPL *session, WT_BLOCK *block)
 {
     WT_CONNECTION_IMPL *conn;
     WT_DECL_RET;
-    uint64_t bucket;
-    u_int i;
+    uint64_t bucket, hash;
 
     conn = S2C(session);
-    bucket = block->name_hash & (conn->hash_size - 1);
+
+    __wt_verbose(session, WT_VERB_BLOCK, "remove: %s", block->name == NULL ? "" : block->name);
```

Comment:
The wording is a bit inconsistent. The function name got changed to "block_close", but the verbose message says that we are removing the handle.

Reply:
Fixed in 6ced72286.